clients/spark/core/src/main/scala/io/treeverse/clients/GCBackupAndRestore.scala

```diff
@@ -0,0 +1,165 @@
+package io.treeverse.clients
+
+import org.apache.hadoop.conf.Configuration
+
+import scala.util.control.Breaks._
+import org.apache.hadoop.tools.DistCp
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.sql.{DataFrame, Dataset, SparkSession}
+
+import java.net.URI
+
+class GCBackupAndRestore {}
+
+object GCBackupAndRestore {
```

Comment:
Documentation?

Reply:
Unfortunately I don't have time to address this comment this week, will open an issue to track it :)